### DISCUSSION TITLE
Bad optimization on old phones.

### DISCUSSION THREAD

**Discussion by Vovancheg228** [ppy/osu#15674] [Q&A] [answered]
Bad optimization on old devices, I play on my phone (Xiaomi redmi note 4x), optimization is terrible (minimum settings), 30 fps in osu! Mania, freezes and lags, in osu!  20-30 fps, also friezes and lags, the phone is normal, the optimization is fig.

**Comment by bdach** [comment, marked as answer]
we're aware.

https://github.com/ppy/osu/search?q=phone+fps&state=closed&type=issues

optimisation is a constant work in progress and mobile devices are currently not a top priority. the situation will be gradually improving but for now the focus is desktop.
